content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+illustrates the options for two clusters. Large systems may employ more
+than one of these options for meshes that span many clusters. See
+[deployment models](/docs/ops/deployment/deployment-models) for more
+information.
+
+{{< tabset category-name="install-istio" >}}
+
+{{< tab name="Multi-Primary" category-value="multi-primary" >}}
+
+The steps that follow will install the Istio control plane on both `FRED` and
+`BARNEY`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Both
+clusters reside on the `EAST` network.
+
+Each control plane observes the API Servers in both clusters for endpoints.
+Service workloads communicate directly (pod-to-pod) across cluster boundaries.
+
+{{< image width="75%"
+    link="multi-primary.svg"
+    caption="Multiple primary clusters on the same network"
+    >}}
+
+### Configure `FRED` as a primary
+
+Install Istio on `FRED`:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_FRED} -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: FRED
+      network: EAST
+      meshNetworks:
+        EAST:
+          endpoints:
+          - fromRegistry: FRED
+          - fromRegistry: BARNEY
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+### Configure `BARNEY` as a primary
+
+Install Istio on `BARNEY`:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_BARNEY} -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: BARNEY
+      network: EAST
+      meshNetworks:
+        EAST:
+          endpoints:
+          - fromRegistry: BARNEY
+          - fromRegistry: FRED
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+### Enable Endpoint Discovery
+
+Install a remote secret in `BARNEY` that provides access to `FRED`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+    --context=${CTX_FRED} \
+    --name=FRED | \
+    kubectl apply -f - --context=${CTX_BARNEY}
+{{< /text >}}
+
+Install a remote secret in `FRED` that provides access to `BARNEY`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+    --context=${CTX_BARNEY} \
+    --name=BARNEY | \
+    kubectl apply -f - --context=${CTX_FRED}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Multi-Primary, Multi-Network" category-value="multi-primary-multi-network" >}}
+
+The following steps will install the Istio control plane on both `FRED` and
+`BARNEY`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. `FRED` is
+on the `WEST` network, while `BARNEY` is on the `EAST` network.
+
+Both `FRED` and `BARNEY` observe the API Servers in each cluster for endpoints.
```

Comment:
I think this image is wrong, it has istiod directly observing the api server in the other cluster, but if it is on a different network it can't do that right? Is istiod using the E-W gateway to get to the api server in the other cluster? Or is the api server exposed some other way? Maybe we should mention how that part works and any requirements on istiod having access to the api server for other clusters?

Reply:
I think many api servers from cloud providers are publicly accessible through an NLB but we should be explicit about that requirement. Some don't meet that and API servers are private, which means this may not work. And for on-prem Kubernetes installs all bets are off...

Reply:
Yeah that was the assumption .. that API Servers are publicly available. I'll add that to the requirements at the top.